tests/ltp/patches/ltp_fsync_fsync04.patch

```diff
@@ -0,0 +1,46 @@
++ // Patch Description: Tests were failing with kernel panic in loop filesystem.
++ // So modified the tests to use root file system.
+diff --git a/testcases/kernel/syscalls/fsync/fsync04.c b/testcases/kernel/syscalls/fsync/fsync04.c
+index c67fc5692..708b0962e 100644
+--- a/testcases/kernel/syscalls/fsync/fsync04.c
++++ b/testcases/kernel/syscalls/fsync/fsync04.c
+@@ -29,10 +29,12 @@ static void verify_fsync(void)
+ {
+        int fd;
+        unsigned long written;
+-
++       rmdir(MNTPOINT);
++        SAFE_MKDIR(MNTPOINT, MODE);
++        SAFE_MOUNT("/dev/vda", MNTPOINT, "ext4", 0, NULL);
+        fd = SAFE_OPEN(FNAME, O_RDWR|O_CREAT, MODE);
+
+-       tst_dev_bytes_written(tst_device->dev);
++       tst_dev_bytes_written("/dev/vda");
+
+        tst_fill_fd(fd, 0, TST_MB, FILE_SIZE_MB);
+
+@@ -41,9 +43,13 @@ static void verify_fsync(void)
+        if (TST_RET)
+                tst_brk(TFAIL | TTERRNO, "fsync(fd) failed");
+
+-       written = tst_dev_bytes_written(tst_device->dev);
++       written = tst_dev_bytes_written("/dev/vda");
+
+        SAFE_CLOSE(fd);
++       if(tst_umount(MNTPOINT) < 0){
```

Comment:
Space before {, fix indentation of if-block

Reply:
remove the test file used as it persists on the device for next test